Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM vvoronin/ubuntu-tools
```

Comment:
What about using https://hub.docker.com/r/rsrchboy/perl-v5.16/ ? -- That should simplify this a lot 👍